Tests/LibWeb/Text/input/class-accessor-enumerable.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
This should be a test under Libraries/LibJS/Tests instead, which gets run by `test-js`. Probably under Libraries/LibJS/Tests/classes.

Reply:
Done